server/src/main/java/org/opensearch/indices/SystemIndexRegistry.java

```diff
@@ -45,7 +45,7 @@ public class SystemIndexRegistry {
     );
 
     private volatile static String[] SYSTEM_INDEX_PATTERNS = new String[0];
-    volatile static Collection<SystemIndexDescriptor> SYSTEM_INDEX_DESCRIPTORS = Collections.emptyList();
+    volatile static Map<String, Collection<SystemIndexDescriptor>> SYSTEM_INDEX_DESCRIPTORS_MAP = Collections.emptyMap();
```

Comment:
This is a great opportunity to make it ``private`

Reply:
It needs to be package-private so it can be used in the `SystemIndices` class in the same package.

Reply:
nvm, pushing a commit where I add a static `SystemIndexRegistry.getAllDescriptors` which can privately use the value and remove all references outside of SystemIndexRegistry.

Reply:
Made private